framework/src/org/checkerframework/common/signatureinference/SignatureInferenceScenes.java

```diff
@@ -0,0 +1,726 @@
+package org.checkerframework.common.signatureinference;
```

Comment:
Thanks Javier and Mike. As already discussed in an email thread, I have implemented Javier's second suggestion.